### PR TITLE
More Rubocop

### PR DESCRIPTION
**Pull request by bquorning**
cc/ @grosser

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -5,3 +5,36 @@ AllCops:
   Exclude:
     - .git/**/*
     - spec/core/middleware/response/sanitize_response_spec.rb
+
+# Align ends correctly.
+Lint/EndAlignment:
+  AlignWith: variable
```
Comment (grosser) [inline .rubocop.yml]: ❤️

File: .rubocop.yml [hunk 2]
```diff
@@ -5,3 +5,36 @@ AllCops:
...
+# Checks the indentation of hanging closing parentheses.
+Style/ClosingParenthesisIndentation:
+  Enabled: false
+
+# Checks the indentation of the first parameter in a method call.
+Style/FirstParameterIndentation:
+  EnforcedStyle: consistent
+
+Style/HashSyntax:
+  EnforcedStyle: hash_rockets
```
Comment (grosser) [inline .rubocop.yml]: wut ?

File: lib/zendesk_api/resources.rb [hunk 3]
```diff
@@ -819,7 +821,7 @@ def new_record?
 
         def self.display!(client, options)
           new(client, options).tap do |resource|
-            resource.save!(path: resource.path + '/display')
+            resource.save!(:path => resource.path + '/display')
```
Comment (grosser) [inline lib/zendesk_api/resources.rb]: 😿

**Comment by grosser** [comment]
indent/self etc looks good ... just hash_rockets feels like the wrong direction ... maybe allow both and then slowly transition to 1.9 style ...

**Comment by grosser** [comment]
👍 anyway since it's better then before ...

**Comment by bquorning** [comment]
My mistake, I thought we were testing against 1.8, but of course we're testing only as low as 1.9. I'll disallow hashrockets, probably in another PR.

**Pull request merged**